### PR TITLE
[CR] Possibility to fix experimental cyborgs at Rubic's Castle

### PR DESCRIPTION
**Pull request by Vgoloshivskiy**
Added possibility to fix cyborgs at Rubic's

#### Summary
Content "Added a way to fix cyborgs at rubic's castle"
#### Purpose of change
Add easier way to fix cyborgs,
I could do it

In future hopefully before stable 0.H it would be possible to add mission to rescue desparete Hub-01 mercenary who got themselves a bionic through Doc or autodoc and ended up as prototype Cyborg (for example mercenary had alergies to a lot of food or had astma).
This is a way to help them but also push a bit conflict with exodii a bit closer.

#### Describe the solution

Added dialogs and a bit of other content in order for this to work
spawned npc should be of hub-01 mrecenary faction 
#### Describe alternatives you've considered
This is new content so i didn't think about them
#### Testing

Run in dev mode to Rubic with ton of items for selling and inactive bodies and tried saving them -> NPC spawned -> but they probably gonna need some new dialog options;

#### Additional context

**I'm not a native english speaker so i have no idea is my grammar correct or no I will be checking it right now.
It also need translation in Rubic's language cause i have no idea how to make it passable for language**

### CODE REVIEW

File: data/json/npcs/exodii/exodii_merchant_talk.json
Comment (RenechCDDA) [inline data/json/npcs/exodii/exodii_merchant_talk.json:455]: All of this *needs* to be using an item group. `u_spawn_item` can accept item groups [with the `use_item_group` argument. ](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/NPCs.md#trade--items)

Comment (Vgoloshivskiy) [reply data/json/npcs/exodii/exodii_merchant_talk.json]: Done cool idea

**Comment by Vgoloshivskiy** [comment]
I think crucial bit is to add mission to return them to the hub so they can pay you from their stash (payment should be oblivious to the player and can be everything from hub coins and nanofab templates to somesing exotic like artifacts or location of undiscovered dungeon)

**Comment by RenechCDDA** [comment]
I know Erk isn't going to have any time in the near future, so I'm going to go ahead and put this back to ready for review. Sorry about the holdup.

File: data/json/npcs/classes.json
Comment (kevingranade) [inline data/json/npcs/classes.json:657]: Still subject to erk's review for overall fit with exodii intentions,  but bio_metabolics, bio_nanobots, and bio_probability_travel are right out for inclusion here.

Comment (Vgoloshivskiy) [reply data/json/npcs/classes.json]: removed them

**Comment by Vgoloshivskiy** [comment]
There are some failing checks but i m honestly not sure are they related to me or not. I used ChatGpt For Rubic translated text as writing dialogue in non-existing language is something that i m too afraid to do

File: data/json/npcs/exodii/exodii_merchant_itemlist.json
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_itemlist.json:382]:
```suggestion
      { "item": "bio_speed", "prob": 6, "count": 1 },
```

File: data/json/npcs/exodii/exodii_merchant_itemlist.json
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_itemlist.json:370]:
```suggestion
```

File: data/json/npcs/exodii/exodii_merchant_itemlist.json
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_itemlist.json:347]:
```suggestion
```

File: data/json/npcs/exodii/exodii_merchant_itemlist.json
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_itemlist.json:346]:
```suggestion
```

File: data/json/npcs/exodii/exodii_merchant_itemlist.json
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_itemlist.json:343]:
```suggestion
```

File: data/json/npcs/exodii/exodii_merchant_itemlist.json
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_itemlist.json:378]:
```suggestion
```

File: data/json/npcs/exodii/exodii_merchant_talk.json
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_talk.json:524]:
```suggestion
        "text": "[Broken Cyborg] Can you save my buddy who looks a bit rough around the edges, but he still hasn't flatlined?",
```

File: data/json/npcs/exodii/exodii_merchant_talk.json
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_talk.json:535]:
```suggestion
      "//~": "For this one I can pull out their cbm and give any working to you, or I can just take out broken bits so that they will still be able to use the rest of stuff in them.  Alternatively why don't you just give him to me I surely can turn him into quite the useful worker.'",
```

File: data/json/npcs/robofac/robofac_ancilla_npcs/BAR_ENCOUNTER_MERCENARIES/NPC_BEMS.json
Comment (Maleclypse) [inline data/json/npcs/robofac/robofac_ancilla_npcs/BAR_ENCOUNTER_MERCENARIES/NPC_BEMS.json:22]:
```suggestion
    "faction": "your_followers"
```

File: data/json/npcs/robofac/robofac_ancilla_npcs/BAR_ENCOUNTER_MERCENARIES/NPC_BEMS.json
Comment (Maleclypse) [inline data/json/npcs/robofac/robofac_ancilla_npcs/BAR_ENCOUNTER_MERCENARIES/NPC_BEMS.json:33]:
```suggestion
    "faction": "your_followers"
```

**Review by Maleclypse — COMMENTED** [review]
The CBMs I cut are all uniques that should only be spawning nether labyrinths.  I also changed faction to your followers after surgery.  This is looking fairly good to go though once those changes commit.

**Review by kevingranade — CHANGES_REQUESTED** [review]
> I used ChatGpt For Rubic translated text as writing dialogue in non-existing language is something that i m too afraid to do

Please pull the ChatGPT produced content back out. ChatGPT is a copyright infringing tool we want nothing to do with.
By submitting changes you are asserting that you wrote it, if you can't make that assertion, don't submit it.

**Comment by Zireael07** [comment]
To add what Kevin said, it's also highly likely that ChatGPT gave you a mix of existing languages instead of what you wanted.

**Comment by Vgoloshivskiy** [comment]
> > I used ChatGpt For Rubic translated text as writing dialogue in non-existing language is something that i m too afraid to do
> 
> Please pull the ChatGPT produced content back out. ChatGPT is a copyright infringing tool we want nothing to do with. By submitting changes you are asserting that you wrote it, if you can't make that assertion, don't submit it.

Will do tommorow can't right now

**Comment by Vgoloshivskiy** [comment]
> > > I used ChatGpt For Rubic translated text as writing dialogue in non-existing language is something that i m too afraid to do
> > 
> > 
> > Please pull the ChatGPT produced content back out. ChatGPT is a copyright infringing tool we want nothing to do with. By submitting changes you are asserting that you wrote it, if you can't make that assertion, don't submit it.
> 
> Will do tommorow can't right now

Done

File: data/json/npcs/exodii/exodii_merchant_itemlist.json
Comment (kevingranade) [inline data/json/npcs/exodii/exodii_merchant_itemlist.json:380]: Auditing the list of available CBMs, many of these were too exotic and some were just kinda dumb.
```suggestion
      { "item": "bio_blood_anal", "prob": 4, "count": 1 },
      { "item": "bio_blood_filter", "prob": 5, "count": 1 },
      { "item": "bio_cable", "prob": 30, "count": 1 },
      { "item": "bio_climate", "prob": 10, "count": 1 },
      { "item": "bio_dex_enhancer", "prob": 8, "count": 1 },
      { "item": "bio_ears", "prob": 17, "count": 1 },
      { "item": "bio_sonar", "prob": 2, "count": 1 },
      { "item": "bio_evap", "prob": 8, "count": 1 },
      { "item": "bio_eye_enhancer", "prob": 2, "count": 1 },
      { "item": "bio_eye_optic", "prob": 2, "count": 1 },
      { "item": "bio_fitnessband", "prob": 16, "count": 1 },
      { "item": "bio_geiger", "prob": 18, "count": 1 },
      { "item": "bio_gills", "prob": 10, "count": 1 },
      { "item": "bio_ground_sonar", "prob": 9, "count": 1 },
      { "item": "bio_heat_absorb", "prob": 4, "count": 1 },
      { "item": "bio_heatsink", "prob": 6, "count": 1 },
      { "item": "bio_radio", "prob": 11, "count": 1 },
      { "item": "bio_sunglasses", "prob": 11, "count": 1 },
      { "item": "bio_ar", "prob": 22, "count": 1 },
      { "item": "bio_alarm", "prob": 26, "count": 1 },
      { "item": "bio_painkiller", "prob": 12, "count": 1 }
```

File: data/json/npcs/classes.json
Comment (kevingranade) [inline data/json/npcs/classes.json:665]: Auditing for exotic or weird CBMs.
```suggestion
```

File: data/json/npcs/classes.json
Comment (Maleclypse) [inline data/json/npcs/classes.json:658]:
```suggestion
      { "id": "bio_power_storage_mkII", "chance": 90 }
```

File: data/json/npcs/exodii/exodii_merchant_talk.json
Comment (Maleclypse) [inline data/json/npcs/exodii/exodii_merchant_talk.json:519]:
```suggestion
        "text": "[Prototype Cyborg] Can you help this guy it seems like they have lost their mind?",
```

**Pull request merged**
